ios/HeapSettings.bundle/generate_settings_common.rb

```diff
@@ -28,3 +39,7 @@ def get_app_id_for_config(specified_config, default_config)
 
   return specified_config['heapAppId'] || default_config['heapAppId'] if is_auto_init
 end
+
+def get_enable_autocapture_for_config(specified_config, default_config)
```

Comment:
nit: `specified` is a bit vague on its own here; consider something like `config_for_selected_environment` instead.